vms/platformvm/block/builder/builder_test.go

```diff
@@ -489,14 +504,16 @@ func TestPreviouslyDroppedTxsCanBeReAddedToMempool(t *testing.T) {
 	require.ErrorIs(reason, errTestingDropped)
```

Comment:
This test is wrong now.

Reply:
Just pushed a fix, i just realized CI was breaking on this since we're now not marking the tx as dropped if it's not in the mempool